docs/uml/macros.iuml

```diff
@@ -307,13 +304,19 @@ skinparam defaultMonospacedFontName Andale Mono
         + hash : BlockHash
       }
 
+      resource(NetworkGenealogy) {
+        + ancestor: Network
+        + descendant: Network
+        ..
+        hashIdField(ancestor, descendant)
+      }
+
+      Network  "2" ---o  NetworkGenealogy
```

Comment:
Since this seems to have gotten messed up in the merge (I want to see how it looks in the preview)

```suggestion
      NetworkGenealogy o--  "2" Network
```